app/views/messages/new.html.erb

```diff
@@ -0,0 +1,9 @@
+
+<%= form_for @message, url: {action: "create"}, html: {class: "nifty_form"} do |f| %>
+ <%= f.label :message %>
+  <%= f.text_field :content %>
+  <% hidden_field :message, :user_id, :value => session[:user_id] %>
```

Comment:
You almost never want to get the user id out of a hidden field on a form on the browser. That means that anyone submitting this form can use the Chrome Developer Tools to change the value of this field, submit the form and post as a different user.

User IDs should almost exclusively come from the currently logged in user via the session.